GetColorCode.c

```diff
@@ -0,0 +1,23 @@
+#include "ColorCode.h"
+
```

Comment:
The name of the file `GetColorCode` - Get is a generic verb... how about something more specific like `TranslateColorCode`?